src/app/shared/header/header.component.ts

```diff
@@ -119,6 +119,18 @@ export class HeaderComponent implements OnInit, OnDestroy {
     });
   }
 
+  public getRouterLink(): string {
+    if (this.isUBS) {
+      if (this.isAdmin) {
+        return '/ubs-admin/orders';
+      } else {
+        return '/ubs';
+      }
+    } else {
+      return '/';
+    }
+  }
```

Comment:
don't use nested if statement

if (this.isUBS && this.isAdmin) {
return '/ubs-admin/orders';
}
if (this.isUBS && !this.isAdmin) {
return '/ubs';
}
return '/';